src/main/java/net/lingala/zip4j/tasks/AddFolderToZipTask.java

```diff
@@ -41,13 +41,13 @@ protected long calculateTotalWork(AddFolderToZipTaskParameters taskParameters) t
     return calculateWorkForFiles(filesToAdd, taskParameters.zipParameters);
   }
 
-  private void setDefaultFolderPath(AddFolderToZipTaskParameters taskParameters) {
+  private void setDefaultFolderPath(AddFolderToZipTaskParameters taskParameters) throws IOException {
```

Comment:
Is it necessary to also change this method? I think just changing it above in `AbstractExtractFileTask.java` is enough?

Reply:
This is necessary. Please refer to my following comment.